### PR TITLE
✨ 31 POST event & POST event/attendance

### LINKED ISSUE #31: Create Event & Attendance

### PR DESCRIPTION
**Pull request by Junnyyy**
What type of PR is this? 🚧
- [x] ✨ Feature

Describe your PR request 🔥
This pull request adds POST to the event and event attendance tables.

Endpoints can be tested using the following:
http://localhost:5050/event
http://localhost:5050/event/attendance

Related Issues 🩹
Resolves #31

### CODE REVIEW

File: src/services/event-service.ts [hunk 1]
```diff
@@ -1,12 +1,84 @@
...
+		const attendance = await prisma.event_attendance.findFirst({
+			where: {
+				contact: {
+					uh_id,
+				},
+				event_id,
+			},
+		});
+
+		if (attendance) {
```
Comment (crudemandarin) [inline src/services/event-service.ts:46]: This if/else block could be optimized with a simple:
```
return !!attendance;
```

File: src/services/event-service.ts [hunk 2]
```diff
@@ -1,12 +1,84 @@
...
+			},
+		});
+
+		return event;
+	}
+
+	static async checkAttendance(uh_id: string, event_id: string) {
+		const attendance = await prisma.event_attendance.findFirst({
+			where: {
+				contact: {
```
Comment (crudemandarin) [inline src/services/event-service.ts:39]: Cool use of nested selection!

**Comment by crudemandarin** [comment]
Looks great! Thanks for the good work Johnny 😄

File: src/services/event-service.ts (hunk 1 above)
Comment (Junnyyy) [reply src/services/event-service.ts:46]: I didn't know that was possible. I will definitely use it in the future.

**Pull request merged**